ROMFS/px4fmu_common/init.d/4250_teal

```diff
@@ -62,36 +53,43 @@ then
 	param set CBRK_USB_CHK 197848
 
 	# commander
-	param set COM_DISARM_LAND 1
-	param set COM_LOW_BAT_ACT 3
-	param set COM_LOSS_LTR_T 20
+	param set COM_DISARM_LAND 0.1 # default -1 (disabled)
+	param set COM_LOW_BAT_ACT 3 # Return mode at critically low level, Land mode at current position if reaching dangerously low levels.
 
 	# ekf2
 	param set EKF2_AID_MASK 1
+	param set EKF2_MAG_TYPE 1
 	param set EKF2_GPS_CHECK 511
 	param set EKF2_GPS_POS_X -0.04
 	param set EKF2_IMU_POS_X -0.06
-	param set EKF2_MIN_RNG 0.07
-	param set EKF2_PCOEF_XN 0.1
-	param set EKF2_PCOEF_XP -0.5
+
+	param set EKF2_PCOEF_XN 0.1 # Static pressure position error coefficient for the negative X axis.
```

Comment:
These comments won't be pruned, so it's better to add them on a new line

Reply:
Only new line comments are pruned? So I should bump all comments to a new line?

Reply:
Right